Furball.Engine/Engine/Graphics/Drawables/UiElements/UiButtonDrawable.cs

```diff
@@ -0,0 +1,52 @@
+using Furball.Engine.Engine.Graphics.Drawables.Managers;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using MonoGame.Extended;
+
+namespace Furball.Engine.Engine.Graphics.Drawables.UiElements {
+    public class UiButtonDrawable : ManagedDrawable {
```

Comment:
Kind of a general thing, but wouldnt it be nice to have the button onhover make the button slightly like brighter? as to indicate that its clickable and stuff, and onhoverlost would make it go back to the original color?, also i would add a Clickable bool that would allow the user to make the button unclickable

Reply:
Whether or not something is clickable should be handled in BaseDrawable, so ill do that outside of the PR, but ill add a small effect onhover rq

Reply:
> Kind of a general thing, but wouldnt it be nice to have the button onhover make the button slightly like brighter? as to indicate that its clickable and stuff, and onhoverlost would make it go back to the original color?, also i would add a Clickable bool that would allow the user to make the button unclickable

resolved in aebeb1c2c28a6f1c8b2f3b1dcf415cf0c7683686